addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -2739,6 +2739,19 @@ var SnippetsMenu = Widget.extend({
             forceEmptyTab: true,
         });
     },
+    /**
+     * Activate the Blocks tab.
+     *
+     * @private
+     */
+    _activateBlocksTab() {
+        this._activateSnippet(false).then(() => {
+            this._updateRightPanelContent({
+                content: [],
+                tab: this.tabs.BLOCKS,
+            });
+        });
```

Comment:
I know it was like that before but is it actually necessary to call `_updateRightPanelContent` after `_activateSnippet` ? Is not this already done by `_activateSnippet` ?

Reply:
Indeed, so we no longer need the new  `_activateBlocksTab` function that I previously created in this commit.. I just added a comment to explain that `this._activateSnippet(false)` activates the Blocks tab.